### PR TITLE
Added "Foundations" Group to Storybook

### PR DESCRIPTION
**Pull request by joncursi**
The Foundations group allows us to visualize, at a deeper level, the underlying values that are set on each theme.

---

Palette Visualizer:

palette

Shadows Visualizer:

shadows-min

Shape Visualizer:

shape

Spacing Visualizer:

spacing

Typography Visualizer:

typography

### CODE REVIEW

**Comment by justinwatkinsact** [comment]
@joncursi , I'm not sure disabling reactDocGen is a good tradeoff.  This is what enables very rich docs with all of the properties, types and comments as discriptions.  That seems like a very big loss for the documentation.  I'm assuming some incompatibility with foundation?

**Comment by joncursi** [comment]
> I'm assuming some incompatibility with foundation?

Hmm, I couldn't get the storybook to run on the `main` branch after a fresh `rm -rf node_modules package-lock.json && npm install`. Turning this off was the only thing that worked for me.

Are you able to reproduce the issue on your side on `main`?

File: .storybook/preview.js [hunk 1]
```diff
@@ -49,7 +49,9 @@ export const parameters = {
     storySort: {
       order: [
         'DLS',
-        'Material UI',
+        'Foundations',
+        'MUI Core',
```
Comment (joncursi) [inline .storybook/preview.js:53]: Decomposed `Material UI` grouping into `MUI Core` and `MUI X` to align with the separation of the two libraries upstream.

**Review by justinwatkinsact — APPROVED** [review]

**Comment by joncursi** [comment]
Got approval from Justin and Vicky (offline), so merging!

**Pull request merged**